lib/Bugsnag.js

```diff
@@ -114,8 +111,8 @@ export class Client {
     let type = metadata['type'] || 'manual';
     delete metadata['type'];
     NativeClient.leaveBreadcrumb({
-      name: name,
-      type: type,
+      name,
+      type,
       metadata: typedMap(metadata)
```

Comment:
I think this is because it can't be shortened like the others since it needs to go through the `typedMap` function.